src/utils/logger.test.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { Logger } from './logger'
+import { Settings } from 'luxon'
+
+describe('Logger', () => {
+  let consoleMock: jest.SpyInstance
+  let originalNow: () => number
+  let dateShift = 0
+
+  function mockConsole(methodToMock: 'error' | 'warn' | 'debug' | 'info', onResult: (result: string) => void) {
```

Comment:
You could use the `LogLevel` enum here instead and make that to a String-enum instead of an Integer-enum.